actors.md

```diff
@@ -162,10 +166,10 @@ type StorageMarketActor struct {
 
 | Index     | Method Name       |
 | -------- | ---------- |
-| 0   | `CreateStorageMiner`     |
```

Comment:
why this change to `1` index?

Reply:
I'm wanting to reserve the 0 value for the constructor